.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+bazel-*
+.idea
```

Comment:
Not a necessary for this PR but we will probably want to add `.ijwb` as well

Reply:
Ah yeah, perhaps open an issue for updating the .gitingore